concepticondata/conceptlists/Hwang-2021-60.tsv-metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+    "@context": [
+        "http://www.w3.org/ns/csvw",
+        {
+            "@language": "en"
+        }
+    ],
+    "dialect": {
+        "encoding": "utf-8-sig",
+        "delimiter": "\t",
+        "skipBlankRows": true
+    },
+    "tables": [
+        {
+            "tableSchema": {
+                "columns": [
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[a-zA-Z]+\\-[0-9]{4}\\-[0-9]+[a-z]?\\-[0-9]+[a-z]?$"
+                        },
+                        "name": "ID"
+                    },
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[0-9\\.]+([a-z\\\u2013]+)?$"
+                        },
+                        "name": "NUMBER"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ENGLISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "CONCEPTICON_GLOSS"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "KOREAN"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "SEMANTIC_CATEGORY"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "SEMANTIC_CATEGORY_TRANSLATED"
```

Comment:
Just one more minor change: I'd use SEMANTIC_CATEGORY_ENGLISH so that it's clear which language.